lib/public/components/Filters/RunsFilter/eorReasonsFilterModel.js

```diff
@@ -0,0 +1,119 @@
+/**
+ * @license
+ * Copyright CERN and copyright holders of ALICE O2. This software is
+ * distributed under the terms of the GNU General Public License v3 (GPL
+ * Version 3), copied verbatim in the file "COPYING".
+ *
+ * See http://alice-o2.web.cern.ch/license for full licensing information.
+ *
+ * In applying this license CERN does not waive the privileges and immunities
+ * granted to it by virtue of its status as an Intergovernmental Organization
+ * or submit itself to any jurisdiction.
+ */
+
+import { RemoteData, Observable } from '/js/src/index.js';
+import { getRemoteData } from '../../../utilities/fetch/getRemoteData.js';
+
+const emptyEorReason = {
+    category: '',
+    title: '',
+    description: '',
+};
+
+/**
+ * Model storing state of a selection of run types picked from the list of all the existing run types
+ */
+export class EorReasonFilterModel extends Observable {
```

Comment:
This file is a class. It should be named with capital letter `EorReasonFilterModel`